packages/odata2model/package.json

```diff
@@ -36,15 +36,15 @@
   "peerDependencies": {
     "@odata2ts/odata-client-api": "*",
     "@odata2ts/odata-query-objects": "*",
-    "@odata2ts/odata-service": "*"
+    "@odata2ts/odata-service": "*",
+    "prettier": "*"
```

Comment:
Using "*" as version isn't a good solution in my opinion, because odata2model needs a specific prettier api interface (Version 2.6.2).

I would either keep the prettier dependency or outsource the prettier formatter into it's own formatter plugin package, which can then be installed and configured separately.

For now, I would leave everything as it is.